### ISSUE TITLE
cljr-slash is triggered when destructuring hashmap namespaced keys

### ISSUE THREAD

**Issue by levitanong** [clojure-emacs/clj-refactor.el#346]
## Expected behavior

``` clj
;; Nothing should happen
(let [{:keys [some/key1 some/key2]} some-map]
  ...)
```
## Actual behavior

``` clj
(let [{:keys [some/ ;; calls cljr-slash
```
## Steps to reproduce the problem
1. Make a map with namespaced keys
2. Make some destructuring of the map using the `:keys` destructuring

``` clj
(def some-map {:foo/bar 1 :foo/baz 2})
;; Just type the below
(let [{:keys [foo/bar foo/baz]} some-map])
```
### clj-refactor.el version information

clj-refactor 2.3.0-SNAPSHOT (package: 20160922.329) nrepl-refactor 2.3.0-SNAPSHOT
### CIDER version information

```
;; CIDER 0.14.0snapshot (package: 20160927.2135), nREPL 0.2.12
;; Clojure 1.9.0-alpha12, Java 1.8.0_51
```
### Leiningen or Boot version

Boot 2.6.0
### Emacs version

emacs-24.5.1
### Operating system

Mac OS Sierra version 10.12

**Comment by benedekfazekas** [comment]
hm #305 seems to be very similar. anyways will have a look

**Comment by levitanong** [comment]
It does seem very similar. I'm working on namespaced maps because I'm writing datomic pull syntax queries.

**Comment by magnars** [comment]
I have this fixed on a local branch. I'll get it merged into the master and push it.

**Comment by magnars** [comment]
Seems like this is actually already fixed in master. See `cljr--in-keyword-sans-alias-p`

**Comment by magnars** [comment]
Doh, I misunderstood the issue here. My issue was already solved by Ben.

**Comment by magnars** [comment]
I never actually solved this one, since we've moved to Clojure 1.9 and use the `:ns/keys` destructuring instead.

**Comment by benedekfazekas** [comment]
ah ok. yeah i guess desctructuring is tricky as it does not use the colon...

**Comment by expez** [comment]
Messed up issue reference in commit message, but fixed in a2523d978025b76d03b5e238e74aa5069f319b9d
